model/test_results.go

```diff
@@ -338,10 +342,11 @@ type TestResult struct {
 // TestResultsFindOptions allows for querying for test results with or without
 // an execution value.
 type TestResultsFindOptions struct {
-	TaskID          string
-	DisplayTaskName string
-	Execution       int
-	EmptyExecution  bool
+	TaskID         string
+	DisplayTaskID  string
+	Variant        string
```

Comment:
If we're removing the ability to search by display task name, shouldn't we also get rid of variant? Or should we keep display task name + variant as an optional means to query data?

Reply:
No, I forgot to delete from an earlier iteration of the code. Removing.